### DISCUSSION TITLE
Homematic-IP-Wettersensor-pro keine Temperaturanzeige in Homeapp-gelöst

### DISCUSSION THREAD

**Discussion by Uwe-F** [thkl/hap-homematic#385] [General]
Ich habe von Homematic den Homematic-IP-Wettersensor-pro mit der Bezeichnung HmIP-SWO-PR. In der Homematic sind viele Variablen erkennbar(siehe Bild).  Aktuell wird die Temperatur mit 0°C angezeigt(Bild), obwohl diese in der CCu angezeigt wird. Luftfeuchtigkeit wird angezeigt. Ist es möglich auch die Temperatur zu übertragen?
![IMG_1298](https://user-images.githubusercontent.com/79526268/109424377-a9e7a600-79e3-11eb-8a6d-fdb3104367c3.jpg)
![Bildschirmfoto 2021-02-23 um 19 25 37](https://user-images.githubusercontent.com/79526268/109424353-95a3a900-79e3-11eb-8d38-536a291bf022.png)
![Bildschirmfoto 2021-02-23 um 19 25 59](https://user-images.githubusercontent.com/79526268/109424359-9a685d00-79e3-11eb-9038-b2be04f760d7.png)

**Comment by thkl** [comment]
Bitte mal in den Einstellungen auf der HAP Seite die Kompatibilität auf Apple Home stellen.
Der Sensor sollte funktionieren laut dem hier  : https://github.com/thkl/hap-homematic/issues/386

**Comment by Uwe-F** [comment]
Die Änderung habe ich vorgenommen, in der Home-App keine Änderung
![Einstellungen](https://user-images.githubusercontent.com/79526268/109491211-93942580-7a89-11eb-8ec4-f8536a11c31f.PNG)
![HAP-Adapter](https://user-images.githubusercontent.com/79526268/109491245-9f7fe780-7a89-11eb-93d4-132f0353b7b6.PNG)
![Temp markiert](https://user-images.githubusercontent.com/79526268/109491272-a6a6f580-7a89-11eb-8f9d-ced6c163499d.PNG)
![Bildschirmfoto 2021-03-01 um 12 19 40](https://user-images.githubusercontent.com/79526268/109491283-a9094f80-7a89-11eb-9c20-076e6e91119c.png)
![Homematic Temp aktuell](https://user-images.githubusercontent.com/79526268/109491566-030a1500-7a8a-11eb-8772-a600bf95af72.PNG)
Die aktuelle Temp. in der Homematic.

**Reply by Uwe-F** [reply]
Einstellung gefunden, jetzt funktioniert es wunderbar. Damit das in der Home App funktioniert muss in den Settings von HAP der Service "HomematicWeatherStationAccessory" eingestellt werden. Das hatte ich übersehen.
